Source/WebCore/dom/EventNames.json

```
@@ -1,5 +1,5 @@
 {
-    "DOMActivate": {"types": ["MouseClickRelated"]},
+    "DOMActivate": { "categories": ["MouseClickRelated"], "defaultEventHandler": true },
```

```suggestion
    "DOMActivate": {
        "categories": ["MouseClickRelated"],
        "defaultEventHandler": true
     },
```
nit: Can we format like this?

---

We can but I don't want each event to take up more than one line.